src/app.module.ts

```diff
@@ -14,8 +15,8 @@ import { users } from './user/user.entity';
       username: 'root',
       password: 'root',
       database: 'learnig_schema',
-      entities: [users],
-      synchronize: true,
+      entities: [users, notes],
```

Comment:
Aby uniknąć każdorazowo deklarowania entity możesz użyć automatycznego ładowania 

entities: ['sciezka do folderu z entity'],

np
entities: [`${baseDir}/**/*.entity{.ts,.js}`],

gdzie dwie gwiazdki ** oznaczają 'dowolny folder' a jedna * dowolną nazwę pliku :)

Reply:
Poprawiłem, Widziałem to wcześniej w dokumentacji ale porzuciłem pomysł gdyż było tam odwołanie do katalogu dist w którym za bardzo nie wiem co się dzieje, a i tak miałem w planie ekstrahować ten obiekt przy okazji rozkminy .dotenv. Co zresztą uczyniłem :)